course/resources/rqm-ecologized.md

```diff
@@ -0,0 +1,227 @@
+- [x] Using other people’s data - intro to datasets
+- [x] Spreadsheet best practices
+- [x] Introduction to metadata
+- [ ] Data creation and authorship
+- [x] Introduction to data cleaning
+- [ ] Grey data liberation
+- [x] Introduction to Scripting
+- [x] Data Cleaning in R
+- [x] Version Control
+- [x] R Review
+- [x] The basics of programming in R
+- [x] Even more with R (writing simple software)
+- [ ] Getting a bit abstract, submit to conference
+- [x] Further topics in R
+- [x] Visual communication with data
+- [ ] Ethics and risks in sharing data
+- [x] Plotting in R
+- [ ] Academic publishing, guest speaker
+- [x] Project progress via github task list
+- [x] Supported work period, Christie away
+- [ ] More on Citizen science approaches, Christie away
+- [ ] Special Guest: Auriel Fournier
+- [x] Supported work period
+- [ ] Open source beehives, Christie away
+- [x] Supported work period
+- [x] Supported work period
+- [ ] Special guest mozilla open science
+- [x] Preparing a paper for publication
+
+### Discussions
+The original RQM syllabus includes lectures, exercises and discussions. While the themes of the exercises largely map well to the lecture topics, the discussions are weighted more toward openness and publishing in general. Below is a list of discussion topics from the RQM course. We need to discuss how many of these we want to include, and if we think the time can be better spent on the lectures and tutorials (this will affect the grading also since some of the blog posts are to reflect on the discussion topics).
+- Openness and reproducibility in research
+    - So why bother with reproducibility? What's the big deal about openness? How do they fit together? in this, the first class discussion, let's think about your motivations for taking this class. We'll begin the discussion by watching the video together. 
+- Authorship in science
+- Grey data liberation
+    - Refer back to Simon Leather’s post on unused data that needs love.
+- Reproducibility and replication
+    - What makes a scientific experiment repeatable? Repoducible? Replicable? Readings and video
+- Licensing
+    - For Tuesday: How do data and content licenses differ? What do you need to keep in mind when assigning a license more restrictive than CC-0 to your work? Talk about what happens if you are integrating multiple data sets and one data set is set as non-commercial while the other is only set as share-and-share alike. Think back to when we discussed MSU's IP policies and revisit what licenses you may or may not be allowed to apply to your work. 
+- Authorship on data, authorship in collaboration
+    - Now that you're a little deeper into the course, you're ready for a more in-depth discussion about scientific authorship. The readings here are meant to present some extreme views and really provoke discussion. Please read these readings prior to class on Thursday.
+- Visualization for communication
```

Comment:
Yes, this topic will be part of the ggplot week. Not sure if we should have a separate discussion on it as well.